src/oca_github_bot/tasks/tag_needs_review.py

```diff
@@ -0,0 +1,28 @@
+from ..github import gh_call, repository
+from ..queue import getLogger, task
+
+_logger = getLogger(__name__)
+
+LABEL_NEEDS_REVIEW = "needs review"
+
+
+@task()
+def tag_needs_review(org, pr, repo, status, dry_run=False):
+    """On a successful execution of the CI tests, adds the `needs review`
+    label to the pull request if it doesn't have `wip:` at the
+    begining of the title (case insensitive). Removes the tag if the CI
+    fails.
+    """
+    with repository(org, repo) as gh_repo:
+        gh_pr = gh_call(gh_repo.pull_request, pr)
+        gh_issue = gh_pr.issue()
+        if status == "success" and not gh_pr.title.lower().startswith("wip:"):
```

Comment:
@Angelfentanez Could you also check for `"[wip]" in gh_pr.title.lower()` as it is frequently used?